islandora_compound_object.module

```diff
@@ -450,18 +470,15 @@ EOQ;
     uasort($results, $sort);
 
     foreach ($results as $result) {
-      $object = islandora_object_load($result['object']['value']);
-      if (islandora_object_access(ISLANDORA_VIEW_OBJECTS, $object)) {
```

Comment:
Not sure this can be removed out right. While XACML falls within the `islandora_object_access` bounds there could be other modules making assertions on viewing.